actions/slack-payload/src/libs/workflow-context/workflow-context.ts

```diff
@@ -72,14 +73,15 @@ const getBaseContext = async (
     runId: workflowRunContext.workflow_run.id,
     sha: workflowRunContext.workflow_run.head_sha.substring(0, 8),
     url: workflowRunContext.workflow_run.html_url,
+    author_email: workflowInfo.author_email || 'Unknown',
```

Comment:
See also my suggestion for getWorkflowInfo
```suggestion
    author_email: workflowInfo.author_email,
```